.github/workflows/auto-update-dev-tools.yml

```diff
@@ -28,9 +28,9 @@ jobs:
         id: create-pr
         with:
           title: '[Auto] Update hlint and fourmolu image'
-          body: 'This PR should be merged automatically.  If it is not merged within 5 minutes, or if all checks have completed, then there is a workflow problem.'
+          body: 'This PR should be merged automatically.  If it is not merged within 30 minutes, or if all checks have completed, then there is a workflow problem.'
```

Comment:
Boy, 5 minutes was really optimistic for doing multiple `cabal install` commands